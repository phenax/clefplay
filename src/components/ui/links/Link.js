
import { h } from 'preact';


export default ({ href, children, ...props }) => (
	<a href={href} cool='wow' {...props}>
		{children}
	</a>
);
