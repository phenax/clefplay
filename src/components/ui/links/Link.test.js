
import { h } from 'preact';
import render from 'preact-render-to-string';

import Link from './Link';

test('Link render', () => {
	const tree = render(
		<Link href='/gogo'>Yo</Link>
	);

	expect(tree).toMatchSnapshot();
});
