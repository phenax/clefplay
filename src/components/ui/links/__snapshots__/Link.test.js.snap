// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link render 1`] = `"<a href=\\"/gogo\\" cool=\\"wow\\">Yo</a>"`;
